skillmap/src/lib/browserUtils.ts

```diff
@@ -282,4 +282,39 @@ export function tickEvent(id: string, data?: { [key: string]: string | number })
     data["pageTitle"] = pageTitle;
 
     (window as any).pxtTickEvent?.(id, data);
-}
\ No newline at end of file
+}
+
+export function hasNavigator(): boolean {
```

Comment:
why copy these methods? pxtlib's browserutils is accessible from skillmap.

Reply:
lol I forgot. i'll fix it